packages/headless/src/controllers/core/generated-answer/headless-core-generated-answer.ts

```diff
@@ -241,7 +245,7 @@ export function buildCoreGeneratedAnswer(
   if (fieldsToIncludeInCitations) {
     dispatch(registerFieldsToIncludeInCitations(fieldsToIncludeInCitations));
   }
-
+  const subscribeStateManager = new SubscribeStateManager();
```

Comment:
Doesn't this create one SubscribeStateManager per controlller and restore the issue we had that led to it's creation intially?
If we have multiple generatedAnswer controllers in the same search interface are they in sync like they should be?

Reply:
Indeed this restores the issues we had when multiple generated answer controllers with the same engine are working in the same time.

Even though the unit tests are passing successfully. But I just found that the unit tests to test this case were not setup properly and thus were passing all the time.